packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.base.tsx

```diff
@@ -82,13 +82,15 @@ export class SearchBoxBase extends React.Component<ISearchBoxProps, ISearchBoxSt
       styles,
       labelText,
       theme,
+      clearable,
```

Comment:
Typically we'd set the default value inline as part of the destructuring (which does the same thing as the code below):
```
clearable = true,
```